var/spack/repos/builtin/packages/r-spacetime/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RSpacetime(RPackage):
+    """spacetime: Classes and Methods for Spatio-Temporal Data"""
+
+    homepage = "http://github.com/edzer/spacetime"
+    url      = "https://cran.r-project.org/src/contrib/spacetime_1.2-2.tar.gz"
+    list_url = "https://cran.r-project.org/src/contrib/Archive/spacetime"
+
+    version('1.2-2', sha256='1e9d3f92e5c4614a69c82a58bf0b5c829b4ed262bf1250bef943ae286056ea2d')
```

Comment:
Can you change this to version `1.2.2`? That would likely make Spack's version parser, etc. happier.
OTOH, it looks like a lot of R packages are like this and I don't know what I'm talking about.

Reply:
This relates to #9455 -- in the end, we decided to keep the original versions with the dashes.